plugins/relay-node-interface/src/index.ts

```diff
@@ -51,7 +58,7 @@ export function relayNodeInterfacePlugin(pluginConfig: RelayNodeInterfacePluginC
             name: 'Node',
             description: 'An object with a global ID',
             definition: (t) => {
-              t.id('id', {
+              ;(nonNullDefaults?.output ? t.nonNull : t.nullable).id('id', {
```

Comment:
Given the default value above, `nonNullDefaults` will never be `null` / `undefined`, so the optional chaining is not needed.